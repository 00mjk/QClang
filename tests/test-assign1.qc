int main() {
    qubit q;
    bit b;
    q = true;
    b = measure(q);
    return 0;
}
