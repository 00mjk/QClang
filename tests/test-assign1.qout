COMPLETED
{'1 0': 1}
dict_keys(['1 0'])
