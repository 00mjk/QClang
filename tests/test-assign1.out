OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
creg c[1];
h q;
qreg temp_0[1];
x temp_0;
creg temp_0_mb1[1];
measure temp_0 -> temp_0_mb1;
